src/shared/common/forms/radio-group.jsx

```diff
@@ -157,28 +157,28 @@ class Radio extends Component {
                                             <label className="govuk-label govuk-radios__label" htmlFor={`${idName}`}>{choice.label}</label>
                                         </div>
                                         {choice.conditionalContent &&
-                                            <div className="govuk-radios__conditional govuk-radios__conditional--hidden"
```

Comment:
Is this a random formatting change that's been introduced? I can't tell the difference... if so, can you remove? Sorry... just easier to maintain the Git History.

Reply:
I have reverted this